lib/internal/test_runner/mock/fake_timers.js

```diff
@@ -3,13 +3,32 @@
 const {
   DateNow,
   SafeMap,
-  Symbol,
+  SafeSet,
   globalThis,
 } = primordials;
 
+const console = require('console')
+const PriorityQueue = require('internal/priority_queue');
+
+function compareTimersLists(a, b) {
+  // console.log({ a, b})
+  const expiryDiff = a.runAt - b.runAt;
+  if (expiryDiff === 0) {
+    if (a.id < b.id)
+      return -1;
+    if (a.id > b.id)
+      return 1;
+  }
+  return expiryDiff;
```

Comment:
```suggestion
  run (a.runAt - b.runAt) || (a.id - b.id);
```
consistent comparators can return any positive or negative number, so this should be sufficient

Reply:
nice!